src/plugins/lightbox/lightbox.js

```diff
@@ -272,7 +277,7 @@ var componentName = "wb-lbx",
 			complete: function() {
 
 				// Set the dependency i18nText only once
-				$.extend( true, $.magnificPopup.defaults, i18nText );
+				$.extend( true, $.magnificPopup.defaults, i18nText, defaults );
```

Comment:
Maybe we should allow the author to extend those setting, like how we do for the data table plugin.

Reply:
Hummm,

After looking how those magnificPopup global default setting is used and after a little local test to override the close button setting, this is not the right place as you pointed out. Thanks for this.

So, I recommend to not change this line, but instead update the following one during the magnificPopup plugin initialization here:

https://github.com/wet-boew/wet-boew/blob/e01bc308c1cc123dbe7948cd8dccc14829ed8e70/src/plugins/lightbox/lightbox.js#L93-L102

Where you can set the default like:

```javascript
 // Extend the settings with window[ "wb-lbx" ] then data-wb-lbx 
 settings = $.extend( 
 	true, 
 	defaults, 
 	settings, 
 	window[ componentName ], 
 	wb.getData( $elm, componentName ) 
 ); 
 $elm.magnificPopup( 
 	settings 
 ).data( "wbLbxFilter", settings.filter ); 
```

---

For your information, I tried to override the setting by configuring the ligthbox button like:

```html
<a class="wb-lbx" href="ajax/ajax4-en.html" data-wb-lbx='{"filter": ".filter-selector", "closeMarkup": "<button type=\u0027button\u0027 class=\u0027mfp-close\u0027><span class=\u0027mfp-close\u0027 aria-hidden=\u0027true\u0027>&times;</span><span class=wb-inv>%title%</span></button>"}'>AJAX - Content filtered by selector (v4.0.12+)</a>
```

Reply:
I see, I retested and I can observe that the close button is not added for the "Gallery" lightbox style.

Having a way to configure that was more a nice to have.